common/src/main/scala/com/gu/recipeasy/models/Recipe.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.recipeasy.models
+
+import java.time.OffsetDateTime
+
+case class Recipe(
+  id: String,
+  body: String,
+  serves: Option[Serves],
+  ingredients: Seq[IngredientList],
+  articleId: String,
+  credit: Option[String],
+  publicationDate: OffsetDateTime,
+  status: Status,
+  steps: Option[List[String]]
+)
```

Comment:
Also we should add `imageUrl: Option[String]`. We can extend the parser to try to find an image for each recipe, as we discussed.